pages/osx/sed.md

```diff
@@ -29,3 +29,7 @@
 - Replace separator / by any other character not used in the find or replace patterns, e.g., #:
 
 `sed 's#{{find}}#{{replace}}#' {{filename}}`
+
+- Remove line number 123 from a file, interactively with no backup:
+
+`sed -i '' '123d' {{filename}}`
```

Comment:
What's `d` means?

Reply:
`d` is for delete

Reply:
What about this: https://github.com/tldr-pages/tldr/pull/4781/files#r510070349:
```suggestion
`sed --in-place='' '{{line_number}}d' {{filename}}`
```

Reply:
@stefanwb wdyt about this?

Reply:
Being more explicit is better indeed 👍

Reply:
How ever this is not a valid option on osx...
```❯ sed --in-place
sed: illegal option -- -
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
And why the example 3 use `--in-place'?

Reply:
That's faulty, it's not supported and should be removed here

Reply:
Oh, yeah I read your original PR title, why you didn't send the deletion/edition of that example then?

Reply:
I've updated both in a new commit